lib/dom-manipulator.js

```diff
@@ -70,6 +55,16 @@ var domManipulator = {
         this.insertTableData(nextLevel, nextLevelPar)
     },
 
+    createParString(par, stroke) {
+      var diff = par - stroke;
+      var absDiff = Math.abs(diff);
+      if (diff === 0) { return "You made par!" };
+      var returnString = "You were " + absDiff;
+      returnString = absDiff === 1 ? returnString + " stroke" : returnString + " strokes";
+      returnString = diff < 0 ? returnString + " under par" : returnString + " over par";
+      return returnString;
+    },
```

Comment:
This looks much cleaner.  Also, as an outsider looking in I think that it is more readable this way too.  Good job!  🛳 it!